packages/blade/src/components/SideNav/docs/SideNav.stories.tsx

```diff
@@ -196,6 +199,29 @@ const navItemsJSON: NavItemsJSONType[] = [
       },
     ],
   },
+  {
+    type: 'section',
+    title: 'International Payments',
+    items: [
+      {
+        icon: GlobeIcon,
+        title: 'Global',
+        href: '/app/global',
+        trailing: (
```

Comment:
<img width="241" alt="image" src="https://github.com/user-attachments/assets/69010410-8a20-45c7-8b1d-93659a92da44" />

We should call this `titleSuffix` right? trailing is when its right aligned. Similar to terminology on `SideNavLink`